### PR TITLE
Ports the src/models/errors folder of cli to Ts

### PR DESCRIPTION
**Pull request by ajsantander**
Part of solving #483

### CODE REVIEW

File: packages/cli/src/models/errors/ErrorHandler.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Logger } from 'zos-lib';
+import ScriptError from './ScriptError';
+
+const log = new Logger('Error');
```
Comment (jcarpanelli) [inline packages/cli/src/models/errors/ErrorHandler.ts]: I'm still typing constant variables that are not inside functions, as our rule was only not to type function bodies :stuck_out_tongue:

File: packages/cli/src/models/errors/ScriptError.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export default class ScriptError extends Error {
+
+  public stack: any;
+  public cb: () => void;
+
+  constructor({ message, stack }: { message: string, stack: any }, cb: () => void) {
```
Comment (jcarpanelli) [inline packages/cli/src/models/errors/ScriptError.ts]: nit: `stack` will always be a `string`.

File: packages/cli/src/models/files/ZosNetworkFile.ts [hunk 3]
```diff
@@ -374,15 +374,7 @@ export default class ZosNetworkFile {
...
-    {
-      package: proxyPackageName,
-      contract: proxyContractName,
-      address: proxyAddress
-    }: {
-        package: string,
-        contract: string,
-        address: string
-      },
+    { package: proxyPackageName, contract: proxyContractName, address: proxyAddress }: ProxyInterface,
```
Comment (jcarpanelli) [inline packages/cli/src/models/files/ZosNetworkFile.ts]: :heart:

File: packages/cli/src/models/errors/ErrorHandler.ts (hunk 1 above)
Comment (ajsantander) [reply packages/cli/src/models/errors/ErrorHandler.ts]: Yeah, I saw this on your PRs. You are typing this stuff, I'm not. I believe that the code looks prettier with inference in these cases. Your call...

File: packages/cli/src/models/errors/ScriptError.ts (hunk 2 above)
Comment (ajsantander) [reply packages/cli/src/models/errors/ScriptError.ts]: Fixed.

**Pull request merged**